docs/hello_world_model.md

```diff
@@ -7,6 +7,9 @@ We have created this file to demonstrate how to:
 1. Configure the UNet3D implemented in this package
 1. Configure Azure HyperDrive based parameter search
 
-- This model can be trained from the commandline: ../InnerEye/runner.py --model=HelloWorld
-- If you have set up AzureML then parameter search can be performed for this model by running:
-../InnerEye/runner.py --model=HelloWorld --hyperdrive=True
+* This model can be trained from the commandline, from the root of the repo: `python InnerEye/runner.py --model=HelloWorld`
+* If you want to test your AzureML workspace with the HelloWorld model:
+    * Upload to datasets storage account for your AzureML workspace: `Test/ML/test_data/dataset.csv` and
```

Comment:
Can we add the training command here?
Train using the command python InnerEye/ML/ runner.py --model=HelloWorld --azureml=True